extra/remove_space.py

```diff
@@ -77,8 +77,8 @@ def rename_class(current_class_name, new_class_name):
 
 with open('class_list.txt') as f:
     for line in f:
```

Comment:
What if we had a line here:
`    line_list = f.read().splitlines() `
`    for line in line_list:`

Reply:
Yeah, that should work. Thank you for your code, it has saved me a good amount of time in my bachelor's thesis!

Reply:
Great! Do you think you could try with this modification and check if it still works?